audit_exceptions/unstable_allowlist.json

```diff
@@ -3,6 +3,7 @@
   "automysqlbackup": "3.0-rc",
   "aview": "1.3.0rc",
   "ftgl": "2.1.3-rc",
+  "go": "1.18rc1",
```

Comment:
Note: this was added temporarily via https://github.com/stefanb/homebrew-core/pull/2 by @gonzaloserrano and should be removed once the Go 1.18 final is released:
```suggestion
```